app/script/util/TimeUtil.js

```diff
@@ -37,6 +37,12 @@ z.util.TimeUtil = {
   formatDuration: (duration, maximumUnits = 1) => {
     const momentDuration = moment.duration(duration);
     const units = [
+      {
+        plural: z.string.ephemeralUnitsYears,
+        singular: z.string.ephemeralUnitsYear,
+        unit: 'y',
+        value: momentDuration.years(),
+      },
```

Comment:
@AndyLnd If we introduce years we should probably also limit the smaller units.

months -> `value: momentDuration.months() % 12`
weeks -> 'value: ....'